### DISCUSSION TITLE
Horizontal alignment of item lists

### DISCUSSION THREAD

**Discussion by MidFidelity** [shedaniel/RoughlyEnoughItems#1507] [Q & A] [answered]
Hi!
Is there a way to change the horizontal alignment of the item and favorite list?
If I limit the number of columns for these lists they are aligned to the edge of the screen with a gap between the Inventory and the item list. 
Is it possible to change this alignment so that I have i.e. 5 columns of items to the right of my inventory and then a gap to the edge of the screen? 
NEI has that option called called `HorizontalAlignment` in the config but I cannot find it in REI.

**Comment by shedaniel** [comment, marked as answer]
This is being worked on, tracking with #1502
